app/ts/store/account.ts

```diff
@@ -4,20 +4,21 @@ import { ApplicationError } from './errors.js'
 import { useNotice } from './notice.js'
 import { useProviders } from './provider.js'
 import { effect, signal, useSignalEffect } from '@preact/signals'
+import { Address, AddressSchema } from '../schema.js'
 import { getAddress } from 'ethers'
 
-const address = signal<AsyncProperty<string>>({ state: 'inactive' })
+const address = signal<AsyncProperty<Address>>({ state: 'inactive' })
 
 export function useAccount() {
 	const { notify } = useNotice()
 	const provider = useProviders()
-	const { value: query, waitFor } = useAsyncState<string>()
+	const { value: query, waitFor } = useAsyncState<Address>()
 
 	const connect = () => {
 		waitFor(async () => {
 			try {
 				const signer = await provider.browserProvider.getSigner()
-				return getAddress(signer.address)
+				return AddressSchema.parse(getAddress(signer.address))
```

Comment:
we could add helper function that does `AddressSchema.parse(getAddress())`

Reply:
Created issue to add checksum validation for parser #202